packages/core/src/components/table/table.css

```diff
@@ -53,6 +55,7 @@
 /* LIGHT VARIANT */
 
 .hds-table--light {
+  --table-background-color: var(--color-white);
```

Comment:
Some alternative name suggestions: table-content-background-color. Or table-rows-background-color.

Reply:
- [x] rename to --content-background-color